components/Layout/MenuBar.tsx

```diff
@@ -51,7 +51,16 @@ export default function MenuBar({ showMenuBarHandler }: MenuBarProps) {
               </div>
             </div>
             <div className={styles.subMenu}>
-              <div onClick={() => MenuPathHandler('manual')}>매뉴얼</div>
+              <div onClick={() => MenuPathHandler('manual')}>경기 매뉴얼</div>
+              <div
+                onClick={() =>
+                  window.open(
+                    'https://github.com/42organization/42arcade.gg.client#주요-기능-소개'
+                  )
+                }
+              >
+                페이지 소개
+              </div>
```

Comment:
```suggestion
               <div
                onClick={() =>
                  window.open(
                    'https://github.com/42organization/42arcade.gg.client#주요-기능-소개'
                  )
                }
              >
                페이지 가이드
              </div>
              <div onClick={() => MenuPathHandler('manual')}>경기 매뉴얼</div>
```
페이지 알려주는 페이지가 먼저오는게 좋지 않을까요?